BeeKit/Managers/RequestManager.swift

```diff
@@ -39,6 +39,8 @@ public class RequestManager {
             urlWithSubstitutions = urlWithSubstitutions.replacingOccurrences(of: "{username}", with: username)
         }
 
+        logger.debug("\(#function) | \(method.rawValue) | \(urlWithSubstitutions)")
```

Comment:
Again, unrelated change.

As a reviewer it can be hard to know if changes like this are deliberate, or were intended as temporary debugging measures, and weren't cleaned up before opening the PR.

Reply:
It is not necessary for the MR itself though it was deliberate to facilitate development.

I am of the opinion that one should be able to follow log statements back to figure out what went wrong. The logging in the app is not yet to that point.

Reply:
removed from this PR